### DISCUSSION TITLE
How to disable upload feature?

### DISCUSSION THREAD

**Discussion by welcoMattic** [umputun/remark42#1497] [Q&A]
Is there a way to disable upload feature for all users?

**Comment by damienalexandre** [comment]
:+1:  It's already disabled for anonymous users; but it would be nice to be able to disable the upload for all users and even remove the button:

![image](https://user-images.githubusercontent.com/225704/192566449-bb5465f0-6925-436e-9347-5f3ae1563232.png)

I know it's not possible to make everything configurable in such a project, but I see this as a security feature - I don't want people to upload illicit pictures on my servers.
